src/Literature/Literatures.tsx

```diff
@@ -672,4 +672,51 @@ export const Literatures: Record<LiteratureName, Literature> = {
     filename: LiteratureName.TheSecretWar,
     text: <Typography></Typography>,
   }),
+  [LiteratureName.BeyondCyberpsychosis]: new Literature({
+    title: "Beyond Cyberpsychosis",
+    filename: LiteratureName.BeyondCyberpsychosis,
+    text: (
+      <Typography>
+        Human augmentation has come a long way since the first prosthetics and implants. Now any and all parts of the
+        human body are able to be improved by technology: strength, speed, perception, intelligence. A large variety of
+        powerful augmentations have been developed, but for most they may as well not exist. How did things end up like
+        this?
+        <br />
+        <br />
+        Part of the answer is, unfortunately, that the exact technology that has allowed us to come so far in human
+        augmentation is also responsible for it's great expense and exclusivity.
+        <br />
+        The early days of human augmentation were plagued with issues. Immunosupressants were needed to keep people's
+        bodies from rejecting the foreign bodies we were implanting, with the body constantly trying to "restore" itself
+        to it's natural state. On top of that, with the messy nature of biology any replacement for a natural organ
+        would fall short of full functionality. Your arm bones don't just provide structural support and anchors for
+        muscles, they also help produce the blood in your body in the marrow. Even the fat in the body is a living organ
```

Comment:
```suggestion
        would fall short of full functionality. The skeleton doesn't just provide structural support and anchors for
        muscles, it also helps produce the blood in your body in the marrow. Even the fat in the body is a living organ
```
As above on first person